src/lib/expression/pqp_select_expression.cpp

```diff
@@ -40,6 +40,8 @@ bool PQPSelectExpression::is_nullable() const {
   return _data_type_info->nullable;
 }
 
+bool PQPSelectExpression::is_correlated() const { return !parameters.empty(); }
```

Comment:
I realized that I don't understand how external and prepared-statement parameters are handled here. Would the `parameters` vector contain both?

Reply:
Nope, the parameters vector only contains the correlated parameters.

Reply:
Nope, only the correlated params are in `parameters`.